internal/monitor/monitor.go

```diff
@@ -46,6 +47,11 @@ func (r *Runtime) BootstrapServices(ctx context.Context) error {
 				stopErr := pgExporterContainer.Stop(ctx, r.dockerClient, types.ContainerStartOptions{})
 				r.logger.Error("stopping exporter container", zap.Error(stopErr))
 			}
+
+			if gfContainer != nil {
+				stopErr := gfContainer.Stop(ctx, r.dockerClient, types.ContainerStartOptions{})
+				r.logger.Error("stopping grafana container", zap.Error(stopErr))
```

Comment:
Do we always want to print the `stopErr` value? (even when the error is nil)

Reply:
Right, will clean it up